packages/jest-parallel/README.md

```diff
@@ -0,0 +1,173 @@
+# jest-parallel
+
+Module for executing heavy tasks under forked processes in parallel, by providing a modern interface (`Promise` based), minimum overhead, and sticky workers.
```

Comment:
can we just say "a `Promise` based interface" rather than modern? The word modern only works at one point in time, and will sound dated in a year. Make it timeless.